javascript/display-data.js

```diff
@@ -1,26 +1,46 @@
 //TODO: write the display functions to render the page
 var collection = [];
 
-var data = function(param){
+var Data = function(param){
   this.library = param.library;
   this.libraryURL = param.libraryURL;
   this.category = param.category;
   this.dateTested = param.dateTested;
   this.review = param.review;
 }
 
-data.prototype.display = function(){
-  $('#library-name').append(this.library);
-  $('#library-url').attr('href',this.libraryURL);
-  $('#library-url').append(this.libraryURL);
-  $('#content-review-template').append('<hr>');
+Data.prototype.display = function(){
+  var $template = $('article.template').clone();
+  $template.attr('data-category',this.category).css('color','white');
+  $template.find('h1').css('margin','0px');
```

Comment:
Not a best practice to write CSS inline, wondering if there was a specific reason behind it?